cmd/restic/cmd_snapshots.go

```diff
@@ -163,6 +163,8 @@ func PrintSnapshots(stdout io.Writer, list restic.Snapshots, compact bool) {
 		}
 	}
 
+	tab.Footer = fmt.Sprintf("Amount of snapshots: %d", len(list))
```

Comment:
What about "Number of snapshots"? Or just "%d snapshots"? I think that's clearer

Reply:
Done